verification/curator-service/ui/src/components/new-case-form-fields/LocationForm.tsx

```diff
@@ -115,9 +115,13 @@ export function PlacesAutocomplete(
                 let newOptions = [] as Loc[];
 
                 if (results) {
-                    newOptions = [...newOptions, ...results];
+                    newOptions = results.map((l) => {
+                        return {
+                            query: inputValue,
+                            ...l,
```

Comment:
Does this need `...newOptions` too?

Reply:
no it was defined as an empty array before so it's useless I think